XenModel/Messages.resx

```diff
@@ -2891,6 +2891,15 @@ Do you want to assign these VMs to the schedule '{0}' instead?</value>
   <data name="COLLAPSE_CHILDREN" xml:space="preserve">
     <value>&amp;Collapse Children</value>
   </data>
+  <data name="COMMAND_RESET_VTPM_WARNING" xml:space="preserve">
+    <value>Resetting the vTPM that is attached to a VM to its original state may result in deactivation of security features on the VM. Do you want to continue?</value>

```

Comment:
```suggestion
<value>Resetting the vTPM that is attached to a VM to its original state may result in deactivation of security features on the VM. Do you want to continue?</value>

Reply:
I don't see why this indentation should change.

Reply:
I think this one was a mistake from my part, ignore the comment :)